tests/integration/models/release.spec.js

```diff
@@ -286,6 +286,66 @@ describe('Release Model', function () {
 			});
 		});
 
+		parallel('balena.model.release.finalize()', function () {
+			const createDraftRelease = async function (commit) {
+				const userId = await balena.auth.getUserId();
+				return await balena.pine.post<BalenaSdk.Release>({
+					resource: 'release',
+					body: {
+						belongs_to__application: this.application.id,
+						is_created_by__user: userId,
+						commit: commit,
+						status: 'success',
+						source: 'cloud',
+						is_final: false,
+						composition: '{}',
+						start_timestamp: new Date().toISOString(),
+					},
+				});
+			};
+
+			it('should finalize a release by id', function () {
+				return createDraftRelease('abcdefg').then(function (draftRelease) {
```

Comment:
async await :cry: :grey_question:

Reply:
isn't returning the promise more performant than awaiting in cases like this?  In any case, take a look at the new approach.

Reply:
Afair since node 12 there is ~0 cost in async await vs promises.
In general all new code we write since then is all in async-await, and use promise chaining only in rare cases.
Moreover since this is about tests, imo optimizing for readability rather than perf makes sense (especially since the bottleneck in the sdk tests is the API requests and not the js execution time).